ui/panels-plugin/src/plugins/line-chart/tooltip/tooltip-model.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { FocusedSeriesArray } from '../utils/focused-series';
+
+export const TOOLTIP_MIN_WIDTH = 350;
+
+export const TOOLTIP_MAX_ITEMS = 50;
```

Comment:
Any thoughts on max number of items to show in the tooltip?

Reply:
Yeah, good question. I think this is fine for now, but we may want to talk more about "enhanced" tooltips or legends in the sidebar some more to better handle charts with a ton of series.